tests/unit/test_fsspec_streaming.py

```diff
@@ -24,3 +26,10 @@ def test_fsspec_streaming(self):
         self.assertEqual(len(nwbfile.electrode_groups), 2)
         self.assertEqual(len(nwbfile.electrodes), 1152)
         self.assertEqual(nwbfile.institution, "AIND")
+
+    @unittest.skipIf(not HAVE_FSSPEC, "fsspec not installed")
+    def test_s3_open_with_consolidated_(self):
+        s3_path = "https://dandiarchive.s3.amazonaws.com/zarr/ccefbc9f-30e7-4a4c-b044-5b59d300040b/"
+        with NWBZarrIO(s3_path, mode='r') as read_io:
+            read_io.open()
+            self.assertIsInstance(read_io.file.store, zarr.storage.ConsolidatedMetadataStore)
```

Comment:
The original issue mentions also adding a test to force a regular open without consolidated metadata - is there a reason that's not included for the fsspec streaming?

```python
# Confirm that opening the file IN 'r-' mode indeed forces a regular open without consolidated metadata
with NWBZarrIO(s3_path, mode='r-') as read_io:
    read_io.open()
    self.assertIsInstance(read_io.file.store, zarr.storage.DirectoryStore)
```